packages/inputs/src/Menu.js

```diff
@@ -8,7 +8,7 @@ const isObject = v => {
 
 export const menu = options => input(Menu, options);
 
-export class Menu extends MosaicClient {
+export class Menu extends ActivatableClient {
```

Comment:
I don't think ActivatableClient is needed here, as this class implements the full `Activatable` interface on its own. Typically I'd expect the class to have a top-level JSDoc comment that includes an annotation indicating that the `Menu` class implements the `Activatable` interface.

That said, I'm OK using a base class if it's helpful elsewhere for runtime type-checking (as opposed to duck typing checks for the `activate` method).

Reply:
Another option is to provide an `isActivatable` utility method, perhaps something like this:

```js
/**
 * Test if a value implements the Activatable interface.
 * @param {any} The value to test
 * @returns {value is Activatable} true if activatable, false otherwise
 */
export function isActivatable(value) {
  return typeof value?.activate === 'function';
}
```